subsys/metrics/src/main/java/org/commonjava/indy/metrics/zabbix/cache/event/MetricsZabbixCacheEvent.java

```diff
@@ -0,0 +1,83 @@
+package org.commonjava.indy.metrics.zabbix.cache.event;
+
+import org.commonjava.indy.subsys.datafile.DataFile;
+
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * Created by xiabai on 5/5/17.
+ */
+public class MetricsZabbixCacheEvent
```

Comment:
I'd get rid of this class, to be honest. Events are something we need to be really careful with, mainly because they are so hard to test.